central/node/datastore/datastore_impl.go

```diff
@@ -235,7 +235,8 @@ func (ds *datastoreImpl) deleteNodeFromStore(ctx context.Context, ids ...string)
 			continue
 		}
 		if err := ds.risks.RemoveRisk(deleteRiskCtx, id, storage.RiskSubjectType_NODE); err != nil {
-			return err
+			errorList.AddError(err)
```

Comment:
interesting, I wonder if this was on purpose or just an accident?

Reply:
I believe accident. Furthermore, the pattern is not consistent between two write ops within same block.